src/main/scala/mesosphere/marathon/upgrade/ScalingCalculator.scala

```diff
@@ -0,0 +1,19 @@
+package mesosphere.marathon.upgrade
+
+import mesosphere.marathon.Protos.MarathonTask
+
+object ScalingCalculator {
+
+  def scalingProposition(runningTasks: Set[MarathonTask],
```

Comment:
It's moved. Didn't call it apply though: it seems strange to have a case class with an `apply` method whose parameters cannot be destructured via pattern matching. Having a convenience method called `propose()`, which turns a set of parameters into the expected case class values, is imho better to understand.